### DISCUSSION TITLE
Renaming "Preferences" to "Settings" for Ventura release?

### DISCUSSION THREAD

**Discussion by isametry** [MonitorControl/MonitorControl#1086] [Polls]
As you might know, Apple is changing its terminology from "Preferences" to "Settings" in the upcoming macOS 13 Ventura – not just for System Settings, but apparently [everywhere](https://twitter.com/thecomputerclan/status/1534271169070759941?s=20&t=69a-ajyQh6CPbhSgxNxVrg), including app-specific preferences (so for example "Safari Preferences" will be renamed to "Safari Settings").

Like the good platform citizen it is, should MonitorControl follow this convention and also rename its own preference section to "settings" as well? Vote / discuss!

**Comment by waydabber** [comment]
I vote change!
